src/main/java/duke/helper/Storage.java

```diff
@@ -0,0 +1,74 @@
+package duke.helper;
+
+import duke.tasks.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a storage object which deals with loading, saving the task into the file.
+ *
+ */
+public class Storage {
+    private File file;
+
+
+    public Storage(String filePath) {
+        String[] pathArr = filePath.split("/");
+        String directoryPath = "/" + pathArr[0];
```

Comment:
Its great that you managed to managed to make this work using Strings to represent paths, but I suggest looking into the newer Java Path library if you have the time 😊